sirepo/pkcli/admin.py

```diff
@@ -49,16 +49,18 @@ def _audit_user(uid, proprietary_sim_types):
                 )
 
     def _link_or_unlink_proprietary_files(sim_type, should_link):
-        d = proprietary_code_dir(sim_type)
-        for e in simulation_db.examples(sim_type):
-            b = sim_data.get_class(sim_type).proprietary_lib_file_basename(e)
-            p = simulation_db.simulation_lib_dir(sim_type).join(b)
+        for f in proprietary_code_dir(sim_type).listdir(
+                fil=lambda x: x.check(file=True),
+                sort=True,
+        ):
+            p = simulation_db.simulation_lib_dir(sim_type).join(f.basename)
             if not should_link:
                 pkio.unchecked_remove(p)
-                continue
+                return
```

Comment:
I don't understand why this was changed from continue to return.

Reply:
It shouldn't have been. https://github.com/radiasoft/sirepo/pull/2490/commits/b2b8aee87b4188997b30427644819f981d3d47e2